src/Persistence/Migrations/20230606122412_AddBaseIdToItems.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Crpg.Persistence.Migrations
+{
```

Comment:
Use file-scoped namespace

Reply:
and i pushed ! 
![image](https://github.com/verdie-g/crpg/assets/47090987/843b2c7c-f9a6-4c73-8656-8f79377fd8b7)

Reply:
There is something i'm clearly not understanding there. I removed the nullabe disable and scoped the namespaces then pushed , yet it still shows up like i didn't?!

Reply:
Actually this is not on my side. My last commit was 40 minute before your comment yet your comment was already outdated when you made it

Reply:
I get it now you commented a specific commit , but i did two commit one after the other : 
![image](https://github.com/verdie-g/crpg/assets/47090987/e38893ee-2ae6-4967-bb85-240231d38e6e)